server/src/main/java/org/jboss/as/server/ServerService.java

```diff
@@ -433,6 +433,7 @@ public void undeploy(DeploymentUnit context) {
             }
             String message = ServerLogger.ROOT_LOGGER.unsuccessfulBoot(String.join(" ", messages));
             bootstrapListener.bootFailure(new Exception(message, cause));
+            cause.printStackTrace();
```

Comment:
This needs to be removed as it can cause an NPE and I believe that is the issue with the test failures.

Reply:
rebased and removed this line